core/controllers/admin.py

```diff
@@ -290,7 +290,7 @@ def post(self):
                     '%s.' % (self.user_id, feature_name, new_rule_dicts))
             self.render_json(result)
         except Exception as e:
-            logging.error('[ADMIN] %s', e)
+            logging.exception('[ADMIN] %s', e)
```

Comment:
Do we need to log the stack trace here as per the issue description?

Reply:
@DubeySandeep I'm not sure what a stack trace is here. Can you provide any reference here.

Reply:
The stack trace is automatically added by `exception` so it doesn't need to be added separately.

Reply:
That is only in places where we want to actually reraise the exception and not swallow it.